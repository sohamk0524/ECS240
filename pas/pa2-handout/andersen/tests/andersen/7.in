c Large test case with mix of all statements
p 100 76
s 0 4 -1 39
s 0 1 -1 8
s 0 5 0 1
s 0 4 0 4
s 0 8 -1 38
s 0 7 -1 15
s 0 6 0 7
s 1 4 0 7
s 0 13 -1 96
s 0 11 -1 16
s 0 4 0 11
s 2 1 2 1
s 0 19 -1 27
s 0 18 -1 24
s 0 11 0 18
s 2 7 3 4
s 0 22 -1 90
s 0 23 -1 26
s 0 16 0 23
s 4 1 1 19
s 0 26 -1 28
s 0 28 -1 31
s 0 16 0 28
s 4 8 0 28
s 0 31 -1 69
s 0 33 -1 38
s 0 1 0 33
s 4 11 1 26
s 0 38 -1 59
s 0 36 -1 42
s 0 34 0 36
s 6 7 6 8
s 0 44 -1 57
s 0 41 -1 49
s 0 7 0 41
s 7 7 8 1
s 0 47 -1 44
s 0 48 -1 54
s 0 7 0 48
s 9 4 0 47
s 0 54 -1 93
s 0 53 -1 58
s 0 40 0 53
s 8 13 10 1
s 0 56 -1 46
s 0 59 -1 65
s 0 25 0 59
s 0 56 5 31
s 0 65 -1 10
s 0 63 -1 67
s 0 11 0 63
s 10 13 3 47
s 0 67 -1 54
s 0 66 -1 71
s 0 9 0 66
s 2 59 0 67
s 0 74 -1 64
s 0 75 -1 77
s 0 7 0 75
s 7 38 5 47
s 0 80 -1 38
s 0 78 -1 81
s 0 38 0 78
s 11 22 2 66
s 0 84 -1 76
s 0 85 -1 89
s 0 21 0 85
s 5 56 16 1
s 0 86 -1 66
s 0 89 -1 91
s 0 43 0 89
s 12 28 9 44
s 0 93 -1 11
s 0 92 -1 96
s 0 7 0 92
s 10 44 4 75
