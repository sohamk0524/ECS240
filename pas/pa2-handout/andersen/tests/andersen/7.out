pt 10 10
pt 10 11
pt 10 15
pt 10 16
pt 10 24
pt 10 26
pt 10 28
pt 10 31
pt 10 44
pt 10 49
pt 10 54
pt 10 57
pt 10 59
pt 10 65
pt 10 67
pt 10 69
pt 10 77
pt 10 81
pt 10 93
pt 10 96
pt 11 10
pt 11 11
pt 11 15
pt 11 16
pt 11 24
pt 11 26
pt 11 28
pt 11 31
pt 11 44
pt 11 49
pt 11 54
pt 11 57
pt 11 59
pt 11 65
pt 11 67
pt 11 69
pt 11 77
pt 11 81
pt 11 93
pt 11 96
pt 1 38
pt 13 96
pt 15 10
pt 15 11
pt 15 15
pt 15 16
pt 15 24
pt 15 26
pt 15 28
pt 15 31
pt 15 44
pt 15 49
pt 15 54
pt 15 57
pt 15 59
pt 15 65
pt 15 67
pt 15 69
pt 15 77
pt 15 81
pt 15 93
pt 15 96
pt 16 10
pt 16 11
pt 16 15
pt 16 16
pt 16 24
pt 16 26
pt 16 28
pt 16 31
pt 16 44
pt 16 49
pt 16 54
pt 16 57
pt 16 59
pt 16 65
pt 16 67
pt 16 69
pt 16 77
pt 16 81
pt 16 93
pt 16 96
pt 1 8
pt 18 24
pt 19 27
pt 21 89
pt 22 90
pt 23 26
pt 24 10
pt 24 11
pt 24 15
pt 24 16
pt 24 24
pt 24 26
pt 24 28
pt 24 31
pt 24 44
pt 24 49
pt 24 54
pt 24 57
pt 24 59
pt 24 65
pt 24 67
pt 24 69
pt 24 77
pt 24 81
pt 24 93
pt 24 96
pt 25 10
pt 25 11
pt 25 15
pt 25 16
pt 25 24
pt 25 26
pt 25 28
pt 25 31
pt 25 44
pt 25 49
pt 25 54
pt 25 57
pt 25 59
pt 25 65
pt 25 67
pt 25 69
pt 25 77
pt 25 81
pt 25 93
pt 25 96
pt 26 10
pt 26 11
pt 26 15
pt 26 16
pt 26 24
pt 26 26
pt 26 28
pt 26 31
pt 26 44
pt 26 49
pt 26 54
pt 26 57
pt 26 59
pt 26 65
pt 26 67
pt 26 69
pt 26 77
pt 26 81
pt 26 93
pt 26 96
pt 28 10
pt 28 11
pt 28 15
pt 28 16
pt 28 24
pt 28 26
pt 28 28
pt 28 31
pt 28 44
pt 28 49
pt 28 54
pt 28 57
pt 28 59
pt 28 65
pt 28 67
pt 28 69
pt 28 77
pt 28 81
pt 28 93
pt 28 96
pt 31 10
pt 31 11
pt 31 15
pt 31 16
pt 31 24
pt 31 26
pt 31 28
pt 31 31
pt 31 44
pt 31 49
pt 31 54
pt 31 57
pt 31 59
pt 31 65
pt 31 67
pt 31 69
pt 31 77
pt 31 81
pt 31 93
pt 31 96
pt 33 38
pt 34 42
pt 36 42
pt 38 10
pt 38 11
pt 38 15
pt 38 16
pt 38 24
pt 38 26
pt 38 28
pt 38 31
pt 38 44
pt 38 49
pt 38 54
pt 38 57
pt 38 59
pt 38 65
pt 38 67
pt 38 69
pt 38 77
pt 38 81
pt 38 93
pt 38 96
pt 39 15
pt 39 49
pt 39 54
pt 39 77
pt 39 96
pt 40 58
pt 4 10
pt 4 11
pt 41 49
pt 4 15
pt 4 16
pt 4 24
pt 4 26
pt 4 28
pt 4 31
pt 4 39
pt 43 91
pt 44 10
pt 44 11
pt 44 15
pt 44 16
pt 44 24
pt 44 26
pt 44 28
pt 44 31
pt 4 44
pt 44 44
pt 44 49
pt 44 54
pt 44 57
pt 44 59
pt 44 65
pt 44 67
pt 44 69
pt 44 77
pt 44 81
pt 4 49
pt 44 93
pt 44 96
pt 4 54
pt 4 57
pt 4 59
pt 4 65
pt 4 67
pt 4 69
pt 47 44
pt 4 77
pt 4 81
pt 48 54
pt 49 10
pt 49 11
pt 49 15
pt 49 16
pt 49 24
pt 49 26
pt 49 28
pt 4 93
pt 49 31
pt 49 44
pt 49 49
pt 49 54
pt 49 57
pt 49 59
pt 4 96
pt 49 65
pt 49 67
pt 49 69
pt 49 77
pt 49 81
pt 49 93
pt 49 96
pt 53 58
pt 5 38
pt 54 10
pt 54 11
pt 54 15
pt 54 16
pt 54 24
pt 54 26
pt 54 28
pt 54 31
pt 54 44
pt 54 49
pt 54 54
pt 54 57
pt 54 59
pt 54 65
pt 54 67
pt 54 69
pt 54 77
pt 54 81
pt 54 93
pt 54 96
pt 56 10
pt 56 11
pt 56 15
pt 56 16
pt 56 24
pt 56 26
pt 56 28
pt 56 31
pt 56 44
pt 56 46
pt 56 49
pt 56 54
pt 56 57
pt 56 59
pt 56 65
pt 56 67
pt 56 69
pt 56 77
pt 56 81
pt 56 93
pt 56 96
pt 57 10
pt 57 11
pt 57 15
pt 57 16
pt 57 24
pt 57 26
pt 57 28
pt 57 31
pt 57 44
pt 57 49
pt 57 54
pt 57 57
pt 57 59
pt 57 65
pt 57 67
pt 57 69
pt 57 77
pt 57 81
pt 57 93
pt 57 96
pt 5 8
pt 59 10
pt 59 11
pt 59 15
pt 59 16
pt 59 24
pt 59 26
pt 59 28
pt 59 31
pt 59 44
pt 59 49
pt 59 54
pt 59 57
pt 59 59
pt 59 65
pt 59 67
pt 59 69
pt 59 77
pt 59 81
pt 59 93
pt 59 96
pt 6 15
pt 63 67
pt 6 49
pt 65 10
pt 65 11
pt 65 15
pt 65 16
pt 65 24
pt 65 26
pt 65 28
pt 65 31
pt 6 54
pt 65 44
pt 65 49
pt 65 54
pt 65 57
pt 65 59
pt 65 65
pt 65 67
pt 65 69
pt 65 77
pt 65 81
pt 65 93
pt 65 96
pt 66 71
pt 67 10
pt 67 11
pt 67 15
pt 67 16
pt 67 24
pt 67 26
pt 67 28
pt 67 31
pt 67 44
pt 67 49
pt 67 54
pt 67 57
pt 67 59
pt 67 65
pt 67 67
pt 67 69
pt 6 77
pt 67 77
pt 67 81
pt 67 93
pt 67 96
pt 69 10
pt 69 11
pt 69 15
pt 69 16
pt 69 24
pt 69 26
pt 69 28
pt 69 31
pt 69 44
pt 69 49
pt 69 54
pt 69 57
pt 69 59
pt 6 96
pt 69 65
pt 69 67
pt 69 69
pt 69 77
pt 69 81
pt 69 93
pt 69 96
pt 7 15
pt 74 64
pt 7 49
pt 7 54
pt 75 77
pt 77 10
pt 77 11
pt 77 15
pt 77 16
pt 77 24
pt 77 26
pt 77 28
pt 77 31
pt 77 44
pt 77 49
pt 77 54
pt 77 57
pt 77 59
pt 77 65
pt 77 67
pt 77 69
pt 7 77
pt 77 77
pt 77 81
pt 77 93
pt 77 96
pt 78 81
pt 7 96
pt 80 38
pt 81 10
pt 81 11
pt 81 15
pt 81 16
pt 81 24
pt 81 26
pt 81 28
pt 81 31
pt 81 44
pt 81 49
pt 81 54
pt 81 57
pt 81 59
pt 81 65
pt 81 67
pt 81 69
pt 81 77
pt 81 81
pt 81 93
pt 81 96
pt 8 38
pt 84 76
pt 85 89
pt 86 66
pt 89 91
pt 92 96
pt 93 10
pt 93 11
pt 93 15
pt 93 16
pt 93 24
pt 93 26
pt 93 28
pt 93 31
pt 93 44
pt 93 49
pt 93 54
pt 93 57
pt 93 59
pt 93 65
pt 93 67
pt 93 69
pt 93 77
pt 93 81
pt 93 93
pt 93 96
pt 96 10
pt 96 11
pt 96 15
pt 96 16
pt 96 24
pt 96 26
pt 96 28
pt 96 31
pt 96 44
pt 96 49
pt 96 54
pt 96 57
pt 96 59
pt 96 65
pt 96 67
pt 96 69
pt 96 77
pt 96 81
pt 96 93
pt 96 96
pt 9 71
